src/xcode/ENA/ENA/Source/Models/Task Scheduling/ENATaskScheduler.swift

```diff
@@ -22,14 +22,17 @@ import UIKit
 enum ENATaskIdentifier: String, CaseIterable {
 	// only one task identifier is allowed have the .exposure-notification suffix
 	case primaryBackgroundTask = "exposure-notification"
+	case secondaryBackgroundTask = "fetch-test-results"
 
 	var backgroundTaskScheduleInterval: TimeInterval? {
 		switch self {
 		case .primaryBackgroundTask: return nil
+		case .secondaryBackgroundTask: return nil
```

Comment:
would it be more clear to name the tasks `FetchTestResultBackgroundTask` or something like that instead of primary/secondaryBackgroundTask?

Reply:
Agreed, have updated accordingly!